soh/src/code/z_kankyo.c

```diff
@@ -1459,6 +1459,8 @@ void Environment_DrawLensFlare(GlobalContext* globalCtx, EnvironmentContext* env
         LENS_FLARE_RING,    LENS_FLARE_CIRCLE1, LENS_FLARE_CIRCLE1, LENS_FLARE_CIRCLE1, LENS_FLARE_CIRCLE1,
         LENS_FLARE_CIRCLE1, LENS_FLARE_CIRCLE1, LENS_FLARE_CIRCLE1, LENS_FLARE_CIRCLE1, LENS_FLARE_CIRCLE1,
     };
+    static s32 epoch = 0;
+    epoch++;
```

Comment:
Is the purpose of these kind of statements to disable interpolation for this object? In any case, use u32 rather than s32 when the number can wrap around since it's UB when an s32 overflows.

Reply:
You never resolved this?

Reply:
missed the s32 vs u32 thing when going through it this last time, i'll make a cleanup pr for those rn

Reply:
The first sentence also still applies. Incrementing epoch in the draw function will basically disable interpolation for this object. Don't know if that was intended...